webapp/sources/rudder/rudder-web/src/main/resources/configuration.properties.sample

```diff
@@ -641,6 +641,19 @@ rudder.users.cleanup.sessions.purgeAfter=30 days
 #
 rudder.auth.provider=file
 
+#
+# To disable access to the REST API using user-generated API token
+# via the api-authorizations plugin, you can use change this option to "disabled"
+#
+rudder.auth.userRestToken=enabled
+
+#
+# To disable access to the REST API using user-generated API token
+# via the api-authorizations plugin for users from a specific provider (declared in rudder.auth.provider),
+# you can use set this option to "disabled"
+#
+rudder.auth.file.userRestToken=enabled
```

Comment:
I think the example should be "oidc" because it's the most common case for using it. 
Plus, I think we should comment out these two properties so that an user without the plugin is not let thinking he can enable these options.

Reply:
yes indeed